ERROR:root:An error occurred while processing the NL text! Error: 'local variable 'actor_new' referenced before assignment'
ERROR:root:An error occurred while processing the NL text! Error: 'local variable 'actor_new' referenced before assignment'
ERROR:flask.app:Exception on /signal_map/cz [GET]
Traceback (most recent call last):
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask/views.py", line 88, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
TypeError: get() got an unexpected keyword argument 'language'
ERROR:flask.app:Exception on /signal_map/en [GET]
Traceback (most recent call last):
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask/views.py", line 88, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
TypeError: get() got an unexpected keyword argument 'language'
ERROR:root:NL text cannot be requested!
ERROR:flask.app:Exception on /signal_map/cz [GET]
Traceback (most recent call last):
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask/views.py", line 88, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
TypeError: get() got an unexpected keyword argument 'language'
ERROR:flask.app:Exception on /signal_map/en [GET]
Traceback (most recent call last):
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask/views.py", line 88, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
TypeError: get() got an unexpected keyword argument 'language'
ERROR:root:NL text cannot be requested!
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:flask.app:Exception on /nltext [GET]
Traceback (most recent call last):
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask_restful/__init__.py", line 458, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask/views.py", line 88, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/imitrob/anaconda3/envs/py36/lib/python3.6/site-packages/flask_restful/__init__.py", line 573, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/imitrob/nl_instruction_processing/gherkan/flask_api/resources/nl_text.py", line 18, in get
    response = API_FSA.requestNLText()
  File "/home/imitrob/nl_instruction_processing/gherkan/flask_api/api_fsa.py", line 338, in requestNLText
    cls.removeState(cls.S_RECEIVED_SIGNAL)
  File "/home/imitrob/nl_instruction_processing/gherkan/flask_api/api_fsa.py", line 123, in removeState
    nextState = cls.getState()
  File "/home/imitrob/nl_instruction_processing/gherkan/flask_api/api_fsa.py", line 90, in getState
    state = cls.fromBytes(statefile.read())
  File "/home/imitrob/nl_instruction_processing/gherkan/flask_api/api_fsa.py", line 80, in fromBytes
    return (byte[0] << 8) | byte[1]
IndexError: index out of range
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
WARNING:root:No match found for phrase.
WARNING:root:No match found for phrase.
WARNING:root:No match found for phrase.
WARNING:root:No match found for phrase.
WARNING:root:No match found for phrase.
WARNING:root:No match found for phrase.
WARNING:root:No match found for phrase.
WARNING:root:No match found for phrase.
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
ERROR:root:Failed to detect {} actions. Please make the actors clearer
